src/modules/ekf2/ekf2_main.cpp

```diff
@@ -484,6 +484,25 @@ void Ekf2::task_main()
 			ctrl_state.pitch_rate = _lp_pitch_rate.apply(sensors.gyro_rad_s[1]);
 			ctrl_state.yaw_rate = _lp_yaw_rate.apply(sensors.gyro_rad_s[2]);
 
+			/* Velocity in Body Frame */
+			float velocity[3];
+			_ekf->copy_velocity(velocity);
+			Vector3f v_n(velocity);
+			matrix::Dcm<float> Tnb(q);
+			Vector3f v_b = Tnb * v_n;
```

Comment:
@sverling Is this inteded to be in body frame? Just asking because the position below is in earth frame.